core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -674,7 +674,8 @@ object KafkaConfig {
     "<code>message.max.bytes</code> (broker config) or <code>max.message.bytes</code> (topic config)."
   val ReplicaFetchWaitMaxMsDoc = "max wait time for each fetcher request issued by follower replicas. This value should always be less than the " +
   "replica.lag.time.max.ms at all times to prevent frequent shrinking of ISR for low throughput topics"
-  val ReplicaFetchMinBytesDoc = "Minimum bytes expected for each fetch response. If not enough bytes, wait up to replicaMaxWaitTimeMs"
+  val ReplicaFetchMinBytesDoc = "Minimum bytes expected for each fetch response. If not enough " +
+    "bytes, wait up to replicaFetchWaitMaxTimeMs"
```

Comment:
I think its better to use for consistency dot separated name : `replicaFetchWaitMaxTimeMs` -> `replica.fetch.wait.max.ms`